docs/src/cli/index.md

```diff
@@ -48,23 +48,23 @@ webknossos convert \
 
 # Create downsampled magnifications
 webknossos downsample data/target
-webknossos downsample --layer_name color data/target
+webknossos downsample --layer-name color data/target
 
 # Compress data in-place (mostly useful for segmentation)
-webknossos compress --layer_name segmentation data/target
+webknossos compress --layer-name segmentation data/target
```

Comment:
@markbader According to typer `compress` has no `--layer-name` option. Yes or no?

Reply:
You are right, compress automatically compresses every not compressed file.